src/m365/spo/commands/list/list-remove.ts

```diff
@@ -100,6 +105,10 @@ class SpoListRemoveCommand extends SpoCommand {
         requestUrl = `${args.options.webUrl}/_api/web/lists/GetByTitle('${formatting.encodeQueryParameter(args.options.title as string)}')`;
       }
 
+      if (args.options.recycle) {
+        requestUrl += `/recycle()`;
```

Comment:
We can remove the brackets at the end, no?

Reply:
Sure. I have removed the brackets and tested it. It works